Firestore/Source/Model/FSTDocumentKey.mm

```diff
@@ -116,7 +116,6 @@ + (BOOL)isDocumentKey:(const ResourcePath &)path {
       } else {
         return NSOrderedSame;
       }
-
```

Comment:
Do we want to include this Firestore change as well, or save that for another PR?